frontend/src/views/main/project/Catalog.vue

```diff
@@ -31,26 +31,34 @@
 import { onActivated, onDeactivated, ref, watch } from "vue";
 import { useCatalogStore } from "@/stores/catalog";
 import { storeToRefs } from "pinia";
-import LoadingFullscreen from "@/components/LoadingFullscreen.vue";
 import { useRouter, useRoute } from "vue-router/composables";
 import { schedulePeriodicJob } from "@/utils/job-utils";
+import mixpanel from "mixpanel-browser";
+import LoadingFullscreen from "@/components/LoadingFullscreen.vue";
 
-const router = useRouter();
 const route = useRoute();
+const router = useRouter();
+const catalogStore = useCatalogStore();
 
 let props = defineProps<{
     projectId: number,
     suiteId?: number
 }>();
 
-const catalogStore = useCatalogStore();
-const { catalog } = storeToRefs(catalogStore);
-
 const componentRoute = 'project-catalog';
 const defaultRoute = 'project-catalog-tests';
 
+const { catalog } = storeToRefs(catalogStore);
 const refreshingRef = ref<() => void>();
 
+
+function trackTabChange(name: string) {
+    mixpanel.track("Catalog tab change", {
+        fromTab: route.path.split('/').pop(),
```

Comment:
For me it would be better to use the route name instead as this won't work for subroutes

(ei. /test/{uuid} would result in uuid which can actually be a test, slicing function or transformation function)